src/sync.cpp

```diff
@@ -620,7 +685,7 @@ void Sync::addstatecachechildren(uint32_t parent_dbid, idlocalnode_map* tmap, st
 
         l->parent_dbid = parent_dbid;
         l->size = size;
-        l->setfsid(fsid);
+        l->setfsid(fsid, l->sync->client->fsidnode);
```

Comment:
lost check (l->sync) != null included in former `setfsid`
Might not be required though